# current agent ID
int32 agent_id
string frame_base_name
---
# ROS time to wait for swarm synchronization
time sync_time
# new unique agent ID provided by the Ground Station if the current one is 0
int32 new_id